cf/developer-access-group.yaml

```diff
@@ -52,26 +52,15 @@ Resources:
                   - "sns:List*"
                 Resource: "*"
 
-        - PolicyName: "can-access-s3-aws-console"
-          PolicyDocument:
-            Version: "2012-10-17"
-            Statement:
-              - Effect: Allow
-                Action: 's3:ListAllMyBuckets'
-                Resource:
-                  - "arn:aws:s3:::*"
-
-        - PolicyName: !Sub "can-manage-${Environment}-s3"
+        - PolicyName: !Sub "can-manage-old-${Environment}-s3"
           PolicyDocument:
             Version: "2012-10-17"
             Statement:
               - Effect: Allow
                 Action: 's3:*'
                 Resource:
                   - !Sub "arn:aws:s3:::*-${Environment}/*"
-                  - !Sub "arn:aws:s3:::*-${Environment}-${AWS::AccountId}/*"
```

Comment:
Doing this would temporarily remove **all** permissions to the new S3 buckets. Is that ok?